### PR TITLE
Fixed administrator management bugs

### PR DESCRIPTION
**Pull request by harriete**

### CODE REVIEW

File: php/static/stores/badges.js [hunk 1]
```diff
@@ -46,26 +46,6 @@ define(function(require) {
...
-        type: 'put',
-        url: '/badges/' + data.id,
-        data: badgeToUpdate
-      }).then(function(badgeData){
-        $.extend(badgeToUpdate, badgeData);
-        store.trigger('change', [store.data]);
-      },function(){
-
-      });
-    },
```
Comment (fskreuz) [inline php/static/stores/badges.js]: Nganu gi remove man ni?

File: php/static/stores/badge-current.js [hunk 2]
```diff
@@ -7,30 +7,23 @@ define(function(require) {
...
     getState: function() {
       return this.data;
     },
-    recalculateCurrentBadge: function(){
-      var badgeId = this.badgeId;
-      this.data = this.badges.filter(function(badge){
-        return badge.id == badgeId;
-      })[0] || {};
-      this.trigger('change', [this.data]);
-    }
```
Comment (fskreuz) [inline php/static/stores/badge-current.js]: Why???

**Comment by fskreuz** [comment]
Please rebase with the latest code ky you're removing existing code.

File: php/static/stores/badge-current.js (hunk 2 above)
Comment (harriete) [inline php/static/stores/badge-current.js]: hala i didn't touch that file. don't apil lang

File: php/static/stores/badges.js (hunk 1 above)
Comment (harriete) [inline php/static/stores/badges.js]: hala i didn't touch that file. don't apil lang

**Pull request closed without merging**